app.rb

```diff
@@ -0,0 +1,123 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def console_entry_point
+    puts 'Welcome to my School Library!'
+    until list_of_options
+      input = gets.chomp
+      if input == '7'
+        puts 'Thank You for using my School Library!'
+        break
+      end
+
+      option input
+    end
+  end
+
+  def list_all_books
+    puts 'no books found! Add a book' if @books.empty?
+    @books.each_with_index { |book, index| puts "#{index} - #{book.title} - Author: #{book.author}" }
+  end
+
+  def list_all_people
+    puts 'no people found! Add a person' if @books.empty?
+    @people.each { |person| puts "#{person.id} - #{person.name} - Age: #{person.age}" }
+  end
+
+  def create_person
+    puts 'To create a student, press 1, to create a teacher, press 2 : '
+    option = gets.chomp
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'invalid input'
+    end
+  end
+
+  def create_student
+    puts 'create student '
+    print 'enter student name: '
+    name = gets.chomp
+    print 'enter student age'
```

Comment:
- [ ] I would recommend you add spacing to the `age` here for a better user experience.